packages/salesforcedx-vscode-soql/.eslintrc.json

```diff
@@ -0,0 +1,11 @@
+{
+  "parser": "@typescript-eslint/parser",
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "extends": ["prettier"],
+  "env": {
+    "es6": true,
+    "node": true
+  }
+}
```

Comment:
@lcampos I wasn't sure what to put in the config or if you have a standard set of linting rules you want to use. I used the same ones here that I got from the `create-lwc-app` with ts for now.

Reply:
Here's the one we're currently using for the newest projects we're working on. Those are not vscode extensions so you might need to tweak it a bit but we're thinking that something very close to this is what we'll introduce in this repo.

https://github.com/forcedotcom/source-deploy-retrieve/blob/develop/.eslintrc.json

Reply:
Cool, I pulled in the rules from the link above, thanks.